recipes-bsp/acpi-tables/samples/edison/arduino.asl

```diff
@@ -23,9 +23,5 @@
  */
 DefinitionBlock ("arduino.aml", "SSDT", 5, "", "ARDUINO", 1)
 {
-    #define MUX_I2C
```

Comment:
No, if you feel this is a need, split either by scheme I proposed, or one used in Galileo

Reply:
I think alternative is to create predefined asl's, one for each case (3 defines = 8 cases).

If you ask me, I would prefer to split out and have a mux.asl, uart.asl, uart4w.asl, spi.asl etc. and load the combination needed for a specific setup at run time. I think there is a 256 limit on SSDT's we can load?

Otherwise, having a single asl, with defines controlled from the machine.conf file (so even outside of recipe) is the next best thing, see [machine/edison.conf](https://github.com/htot/meta-intel-edison/blob/master/meta-intel-edison-bsp/conf/machine/edison.conf) last 2 lines.

Reply:
But this can be created with one ASL source. Just provide as many defines through command line as you want. So, basically we need arduino-no-mux asl and Makefile to handle proposed variants.

Reply:
Yes, using outside of Yocto, with a Makefile would work nicely.
Currently recipe acts as simple Makefile i.e. deletes all targets and build them all new. The recipe would be prettier if it would also call the Makefile. But I'm not really a Makefile wizard and didn't want to touch the recipe too much when I started this.
With a Makefile we could eleminate also from the other platforms unneeded asl's.

Reply:
For now created arduino-all and leave makefile for future improvement.